src/DqtApi/DataStore/Crm/Models/FindTeachersQuery.cs

```diff
@@ -10,7 +10,7 @@ public class FindTeachersQuery
         public string LastName { get; set; }
         public string PreviousFirstName { get; set; }
         public string PreviousLastName { get; set; }
-        public DateOnly? DateOfBirth { get; set; }
+        public DateTime? DateOfBirth { get; set; }
```

Comment:
This should be `DateOnly` - the conversion to `DateTime` should be inside the `DataverseAdapter` method.